### ISSUE TITLE
how to change hologan loss function ?

### ISSUE THREAD

**Issue by andyandytony** [christopher-beckham/hologan-pytorch#6]
i just wanna compare with different loss function, and get some results.
